src/test/resources/org/magicdgs/readtools/utils/mappability/gem/GemMappabilityReaderUnitTest/example.gem.mappability

```diff
@@ -0,0 +1,113 @@
+~~K-MER LENGTH
+100
+~~APPROXIMATION THRESHOLD
+7
+~~MAX MISMATCHES
+4
+~~MAX ERRORS
+4
+~~MAX BIG INDEL LENGTH
+15
+~~MIN MATCHED BASES
+80
+~~STRATA AFTER BEST
+1
+~~ENCODING
+' '~[0-0]
+'!'~[1-1]
+'"'~[2-2]
+'#'~[3-3]
+'$'~[4-4]
+'%'~[5-5]
+'&'~[6-6]
+'''~[7-8]
+'('~[9-11]
+')'~[12-14]
+'*'~[15-18]
+'+'~[19-23]
+','~[24-29]
+'-'~[30-37]
+'.'~[38-47]
+'/'~[48-60]
+'0'~[61-75]
+'1'~[76-95]
+'2'~[96-119]
+'3'~[120-150]
+'4'~[151-188]
+'5'~[189-237]
+'6'~[238-298]
+'7'~[299-374]
+'8'~[375-470]
+'9'~[471-590]
+':'~[591-740]
+';'~[741-929]
+'<'~[930-1167]
+'='~[1168-1465]
+'>'~[1466-1838]
+'?'~[1839-2307]
+'@'~[2308-2896]
+'A'~[2897-3635]
+'B'~[3636-4561]
+'C'~[4562-5725]
+'D'~[5726-7184]
+'E'~[7185-9016]
+'F'~[9017-11315]
+'G'~[11316-14200]
+'H'~[14201-17821]
+'I'~[17822-22364]
+'J'~[22365-28066]
+'K'~[28067-35221]
+'L'~[35222-44200]
+'M'~[44201-55469]
+'N'~[55470-69609]
+'O'~[69610-87355]
+'P'~[87356-109625]
+'Q'~[109626-137572]
+'R'~[137573-172643]
+'S'~[172644-216655]
+'T'~[216656-271887]
+'U'~[271888-341199]
+'V'~[341200-428181]
+'W'~[428182-537337]
+'X'~[537338-674320]
+'Y'~[674321-846224]
+'Z'~[846225-1061951]
+'['~[1061952-1332674]
+'\'~[1332675-1672411]
+']'~[1672412-2098756]
+'^'~[2098757-2633789]
+'_'~[2633790-3305218]
+'`'~[3305219-4147813]
+'a'~[4147814-5205209]
+'b'~[5205210-6532166]
+'c'~[6532167-8197402]
+'d'~[8197403-10287155]
+'e'~[10287156-12909645]
+'f'~[12909646-16200683]
+'g'~[16200684-20330700]
+'h'~[20330701-25513576]
+'i'~[25513577-32017716]
+'j'~[32017717-40179947]
+'k'~[40179948-50422963]
+'l'~[50422964-63277218]
+'m'~[63277219-79408388]
+'n'~[79408389-99651855]
+'o'~[99651856-125055960]
+'p'~[125055961-156936297]
+'q'~[156936298-196943843]
+'r'~[196943844-247150455]
+'s'~[247150456-310156166]
+'t'~[310156167-389223834]
+'u'~[389223835-488448108]
+'v'~[488448109-612967484]
+'w'~[612967485-769230406]
+'x'~[769230407-965329211]
+'y'~[965329212-1211419204]
+'z'~[1211419205-1520244565]
+'{'~[1520244566-1907798333]
+'|'~[1907798334-2394150627]
+'}'~[2394150628-3004488014]
```

Comment:
We should use a range like this in the test to show the problem with integers...